modules/ROOT/pages/asset-details.adoc

```diff
@@ -73,6 +74,37 @@ image::ex2-rest-ftns.png[Screenshot - REST API function buttons in left nav bar]
 This is similar to using a program such as Postman to specify custom HTTP headers and test each API's method using the API's endpoint.
 . To connect the API to one of your applications and use the API to send and receive data, click *Request access*.
 
+== View Conformance Status 
+
+You can view governance conformance status in Exchange for versions of API definitions. The status is one of the following:
```

Comment:
What are the supported asset types for which conformance status is visible? "API definitions" is a little unclear to me - can we call the specific asset types out upfront?